gpflow/utilities/utilities.py

```diff
@@ -77,7 +77,16 @@ def print_summary(module: tf.Module, fmt: str = None):
     Prints a summary of the parameters and variables contained in a tf.Module.
     """
 
-    fmt = fmt if fmt is not None else summary_fmt()
+    if fmt == "notebook":
+        from IPython.core.display import display, HTML
+        tab = tabulate_module_summary(module, "html")
+        display(HTML(tab))
+    else:
+        print(tabulate_module_summary(module, fmt))
+
+
+def tabulate_module_summary(module: tf.Module, tablefmt: str = None) -> str:
+    tablefmt = tablefmt if tablefmt is not None else summary_fmt()
```

Comment:
I would pass through `None` (meaning "use tabulate's default"); someone who wants to use `tabulate_module_summary` directly probably has a specific use-case.